vendor/k8s.io/kubernetes/pkg/registry/rbac/escalation_check.go

```diff
@@ -37,7 +37,10 @@ func EscalationAllowed(ctx genericapirequest.Context) bool {
 	// system:masters is special because the API server uses it for privileged loopback connections
 	// therefore we know that a member of system:masters can always do anything
 	for _, group := range u.GetGroups() {
-		if group == user.SystemPrivilegedGroup {
+		// allow upstream group AND origin admin group to escalate
+		// TODO: remove origin case when origin has migrated to upstream RBAC
+		originSystemPrivilegedGroup := "system:cluster-admins"
```

Comment:
if all you need is to pass the test, bind the `cluster-admins` role to this group in the test setup